src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -602,6 +775,11 @@ public virtual int LastIndexOf(Object value, int startIndex, int count)
 #if FEATURE_NETCORE
         [FriendAccessAllowed]
 #endif
+        /// <summary>Returns a read-only <see cref="T:System.Collections.IList" /> wrapper.</summary>
```

Comment:
These comments should be above attributes (even when conditionally defined), rather than below.